src/components/AddTaskUI.tsx

```diff
@@ -0,0 +1,36 @@
+import { ChangeEvent } from 'react';
+
+interface AddTaskUIProps {
+  isBlank: boolean;
+  handleTaskInput: (e: ChangeEvent<HTMLInputElement>) => void;
+  newTask: string;
+}
+
+function AddTaskUI({ isBlank, handleTaskInput, newTask }: AddTaskUIProps) {
+  return (
+    <>
+      <div className="relative w-full">
+        <input
+          className={`w-full border rounded-md p-2 
+          ${isBlank ? 'border-red-600' : 'border-black'}`}
```

Comment:
Force this kind of thing where you have conditional classes, check out a package like [`clsx` ](https://www.npmjs.com/package/clsx)

Reply:
added clsx: 

`className={clsx('w-full border rounded-md p-2', {
            'border-red-600': isBlank,
            'border-black': !isBlank,
          })}`